Mollier/SAM.Core.Mollier.UI/Modify/AddMollierZones.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SAM.Geometry.Planar;
+using System.Windows.Forms.DataVisualization.Charting;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Modify
+    {
+
+        public static void AddMollierZones(this Chart chart, IEnumerable<MollierZone> mollierZones, MollierControlSettings mollierControlSettings)
```

Comment:
Return Series -> List
Move to SAM.Core.Mollier.Create
Rename to Series. Example below:

namespace SAM.Core.Mollier.UI
{
public static partial class Create
{
public static List Series(this Chart chart, IEnumerable<MollierZone> mollierZones, MollierControlSettings mollierControlSettings)
{
//Add code here
}
}
}